File: src/main/java/pl/daffit/megakilof/command/MegakilofCommand.java
package pl.daffit.megakilof.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import pl.daffit.megakilof.MegaKilofPlugin;

public class MegakilofCommand implements CommandExecutor {

    private final MegaKilofPlugin megaPlugin;

    public MegakilofCommand(MegaKilofPlugin megaPlugin) {
        this.megaPlugin = megaPlugin;
    }

    @Override
    public boolean onCommand(CommandSender megaSender, Command megaCommand, String megaLabel, String[] megaArguments) {

        if (!(megaSender instanceof Player)) {
            megaSender.sendMessage(MegaKilofPlugin.CONSOLE_DENY_MESSAGE);
            return true;
        }

        ((Player) megaSender).openInventory(this.megaPlugin.getMegaInventory());
        return true;
    }
}

File: src/main/java/pl/daffit/megakilof/command/MegakilofCommand.kt
package pl.daffit.megakilof.command

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import pl.daffit.megakilof.MegaKilofPlugin

class MegakilofCommand(private val megaPlugin: MegaKilofPlugin) : CommandExecutor {

    override fun onCommand(megaSender: CommandSender, megaCommand: Command, megaLabel: String, megaArguments: Array<String>): Boolean {

        if (megaSender !is Player) {
            megaSender.sendMessage(MegaKilofPlugin.CONSOLE_DENY_MESSAGE)
            return true
        }

        megaSender.openInventory(this.megaPlugin.megaInventory)
        return true
    }
}
